keyboards/doio/kb38/keymaps/default/rules.mk

```diff
@@ -0,0 +1 @@
+# forgot what goes here
```

Comment:
Were you going to enable encoder map here? If not, this file should just be removed.